src/libstd/keyword_docs.rs

```diff
@@ -1100,10 +1100,31 @@ mod trait_keyword {}
 //
 /// A value of type [`bool`] representing logical **true**.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// "Beauty is truth, truth beauty, — that is all
+/// Ye know on earth, and all ye need to know." - Keats
```

Comment:
It doesn't feel like this is adding anything?

Reply:
It's true, I can't logically justify Poetry. A friend suggested it would be fitting here.